projects/storefrontapp-e2e-cypress/cypress/integration/regression/billing-address.e2e-spec.ts

```diff
@@ -36,7 +36,6 @@ context('Payment billing address', () => {
 
   it('should go to checkout', () => {
     checkout.registerUser();
```

Comment:
Add `verifyGlobalMessageAfterRegistration();` to avoid cancelling register requests by cypress